### PR TITLE
Feature/pixelated integration + packaging fixes

### PR DESCRIPTION
**Pull request by kalikaneko**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -167,6 +167,36 @@ sumo_tarball_latest: checkout_leapdeps_develop pull_leapdeps setup_without_names
 pyinst:
 	pyinstaller -y pkg/pyinst/bitmask.spec
 
+pyinst-hacks:
+	mkdir -p dist/bitmask/cryptography/hazmat/primitives/src/
+	cp ~/src/cryptography-0.9.3/src/cryptography/hazmat/primitives/src/* dist/bitmask/cryptography/hazmat/primitives/src/
```
Comment (meskio) [inline Makefile]: This looks really hacky to me. `~/src/cryptography-0.9.3`? I guess is a left over from some iterations. Can it be removed?

File: pkg/PixelatedWebmail.README [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+SkipFirstRun=true
+Provider=mail.bitmask.net
+Pixmail=true
+
+[mail.bitmask.net]
+Services=mx
+
+Then, run bitmask again:
+
+./bitmask --debug --danger
```
Comment (meskio) [inline pkg/PixelatedWebmail.README]: Maybe we don't need to have the `--danger` flag here.

File: pkg/pyinst/bitmask.spec [hunk 3]
```diff
@@ -33,6 +34,6 @@ if sys.platform.startswith("darwin"):
 		     name=os.path.join(
 		      'dist', 'Bitmask.app'),
                      appname='Bitmask',
-                     version='0.9.0rc2',
+                     version='0.9.2alpha1',
 		     icon='pkg/osx/bitmask.icns',
-		     bundle_identifier='bitmask-0.9.0rc2')
+		     bundle_identifier='bitmask-0.9.2alpha1')
```
Comment (meskio) [inline pkg/pyinst/bitmask.spec]: Maybe one day we can change this hardcoded versions form something generated from the makefile, getting the tag of the current commit and/or adding the commit-id or something there.

File: Makefile (hunk 1 above)
Comment (kalikaneko) [inline Makefile]: not really, it needs more work though. there are some include files that need to be added. might need to be updated, unzipped, and extracted into the right path...

File: pkg/PixelatedWebmail.README (hunk 2 above)
Comment (kalikaneko) [inline pkg/PixelatedWebmail.README]: true, will remove.

File: pkg/pyinst/bitmask.spec (hunk 3 above)
Comment (kalikaneko) [inline pkg/pyinst/bitmask.spec]: yes, I thought on this, but it needs a while to get it right.
the thing is that I have to fake it manually until the tag is done.

File: Makefile (hunk 1 above)
Comment (kalikaneko) [inline Makefile]: uh. it turns out I had an old cryptography version in my venv, that's why I needed the hacks.
newer versions get this shit included properly via pyinstaller hooks.
removing, thanks for catching this.

**Pull request merged**